documentation/asciidoc/accessories/display/display_intro.adoc

```diff
@@ -1,45 +1,86 @@
 == Raspberry Pi Touch Display
 
-The https://www.raspberrypi.com/products/raspberry-pi-touch-display/[Raspberry Pi Touch Display] is an LCD display which connects to the Raspberry Pi through the DSI connector. In recent versions of Raspberry Pi OS, both the HDMI and LCD display outputs can be used at the same time.
+The https://www.raspberrypi.com/products/raspberry-pi-touch-display/[Raspberry Pi Touch Display] is an LCD display which connects to the Raspberry Pi through the DSI connector. If your Raspberry Pi has both connectors, you can use the micro HDMI and LCD display outputs at the same time.
```

Comment:
"LCD display outputs" should be "DSI output"

Reply:
Done